README.md

```diff
@@ -67,7 +90,7 @@ As all the deleters are known at compiler time and don't need to store anything
 To allow using lambdas as callbacks, `GLFWwindow` user pointer is used internally to store `std::function` objects instead of c-style function pointers.
 These objects are accessible through `vkfw::Window::callbacks()` (or `accessWindowCallbacks(GLFWwindow *)` if enhanced mode is disabled) function and can be set directly, for example:
 ```c++
-my_window.callbacks().on_key = [](vkfw::Window const &, vkfw::Key key, int32_t, 
+my_window.callbacks().on_key = [](vkfw::Window const &, vkfw::Key key, int32_t,
```

Comment:
Please, move formatting changes into a separate commit(s), as they have nothing to do with CMake.

Reply:
Wait, where did these changes go? Please include them as a separate commit.

Reply:
Sounds good!